src/wwElement.vue

```diff
@@ -20,6 +20,7 @@
             :autocomplete="content.autocomplete ? 'on' : 'off'"
             :placeholder="isAdvancedPlaceholder ? '' : wwLang.getText(content.placeholder)"
             :style="style"
+            :pattern="content.pattern"
```

Comment:
if type changed after setting a value, you do not see the field anymore but it is still used. Should have a computed here returning it only if the type is correct?